<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoCheckinManager - Config</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>签到服务列表</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="lw_service"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_new_service">
           <property name="text">
            <string>新建服务</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>服务设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="lb_service_id">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lb_service_command">
            <property name="text">
             <string>命令</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="pb_save_service">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_delete_service">
              <property name="palette">
               <palette>
                <active>
                 <colorrole role="ButtonText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>255</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="ButtonText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="ButtonText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>120</red>
                    <green>120</green>
                    <blue>120</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="text">
               <string>删除服务</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lb_service_wait_time">
            <property name="text">
             <string>随机等待时间</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lb_service_name">
            <property name="text">
             <string>名称</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QSpinBox" name="sb_wait_time_min">
              <property name="maximum">
               <number>2147483647</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_to">
              <property name="text">
               <string>至</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sb_wait_time_max">
              <property name="maximum">
               <number>2147483647</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_second">
              <property name="text">
               <string>秒</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lb_service_config_key">
            <property name="text">
             <string>配置文件字段</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_service_id">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="le_service_name"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="le_service_config_key"/>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLineEdit" name="le_service_command"/>
            </item>
            <item>
             <widget class="QToolButton" name="pb_service_command_help">
              <property name="text">
               <string>?</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>用户列表</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListWidget" name="lw_user"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_new_user">
           <property name="text">
            <string>新建用户</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>用户设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="7" column="1">
           <widget class="QPushButton" name="pb_edit_user_config">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>使用编辑器修改</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lb_user_id">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="lb_user_config">
            <property name="text">
             <string>配置文件</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QPushButton" name="pb_delete_user">
            <property name="palette">
             <palette>
              <active>
               <colorrole role="ButtonText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="ButtonText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="ButtonText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>120</red>
                  <green>120</green>
                  <blue>120</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="text">
             <string>删除用户</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lb_user_email">
            <property name="text">
             <string>邮件地址</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="le_user_email"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_user_id">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="le_user_name"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lb_user_name">
            <property name="text">
             <string>名称</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="pb_save_user">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>其它设置</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_email_account"/>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="pb_save_config">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_email_password">
         <property name="text">
          <string>邮箱密码</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_email_server"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_email_server">
         <property name="text">
          <string>邮件服务器</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="le_email_password">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_email_account">
         <property name="text">
          <string>邮箱账户</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lb_notice_type">
         <property name="text">
          <string>用户通知类型</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QRadioButton" name="rb_no_notice">
           <property name="text">
            <string>不通知</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_all_notice">
           <property name="text">
            <string>一起通知</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_div_notice">
           <property name="text">
            <string>独立通知</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>测试</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="cb_skip_wait">
           <property name="text">
            <string>跳过随机等待</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_only_choose_service">
           <property name="text">
            <string>仅执行当前选择服务</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_only_choose_user">
           <property name="text">
            <string>仅执行当前选择用户</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_send_email_notice">
           <property name="text">
            <string>发送邮件通知</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="pb_run_checkin">
           <property name="text">
            <string>立即执行签到</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lb_last_checkin_time_tip">
          <property name="text">
           <string>上次自动签到时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_last_checkin_time">
          <property name="text">
           <string>1970-01-01 08:00:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
